django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -19,14 +19,10 @@
     min-height: 400px;
 }
 
-.change-list .filtered {
-    background: #fff !important;
-}
-
```

Comment:
Don't need this rule because `change-list`'s parent already has white background.